docs/Training-ML-Agents.md

```diff
@@ -179,7 +178,8 @@ use during training, and the answers to the above questions will dictate its con
 The rest of this guide breaks down the different sub-sections of the trainer config file
 and explains the possible settings for each.
 
-**NOTE:** The configuration file format has been changed from 0.17.0 and onwards. To convert
+**NOTE:** The configuration file format has been changed between 0.17.0 and onwards and
+between 0.18.0 and onwards. To convert
```

Comment:
I think it's sufficient to say that it's changed from 0.17.0. 0.18.0 isn't out yet and this implies the configuration file format will change in the future.